extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss

```diff
@@ -14,10 +14,10 @@
 }
 
 .WikiaPage {
-	border: 0;
+	border: 0 !important;
 
 	.WikiaPageBackground {
-		background: none;
+		background: none !important;
```

Comment:
Aren't all of our CSSes (including this one) loaded for all pageviews? This rule might overwrite a bit to much. How are the 'usual' pages looking?